src/components/date-picker/date-picker.tsx

```diff
@@ -250,6 +255,14 @@ export class DatePicker {
         this.nativeType = nativeTypeForConsumerType[this.type || 'default'];
         this.nativeFormat = nativeFormatForType[this.nativeType];
 
+        // Quarter is not a valid type.
+        // It's therefore set to type "text" such that we can set the
```

Comment:
```suggestion
        // It's therefore set to type "text" so that we can set the
```

#### The very short version:

**so that** is _much_ more common than **such that**—when in doubt, stick to **so that**.

#### Short version:

> _So that_ denotes the purpose or the effect of something or it carries the idea ‘in order that’ while _such that_ emphasizes on the manner or the way to which extent something is described.

> He checked carefully, so that the mistakes were caught.

The goal was to catch the mistakes, checking carefully was a means to an end.

> I spoke to my teacher such that she took me to the principal.

The goal wasn't to be taken to the principal, the "I" was just angry and used some impolite language. The (in this case unwanted) result was being sent to the principal.

#### Long version:

https://www.differencebetween.com/difference-between-so-that-and-vs-such-that/ 😄